tests/lib/rules/no-fallthrough.js

```diff
@@ -109,8 +130,128 @@ ruleTester.run("no-fallthrough", rule, {
         {
             code: "switch (a) {\n case 1: ; break; \n case 3: }",
             options: [{ allowEmptyCase: false }]
+        },
+        `
+switch (foo) {
+    case 0:
+        a();
+}
+switch (bar) {
+    case 1:
+        b();
+}
+            `,
+        {
+            code:
+        `
+switch (foo) {
+    case 0:
+        a();
+        break;
+        // falls through
+}
+switch (bar) {
+    case 1:
+        b();
+}
+            `,
+            options: [{ reportUnusedFallthroughComment: true }]
+        },
+        {
+            code:
+        `
+switch (foo) {
+    case 0:
+        a();
+        break;
+        /* falls through */
+}
+switch (bar) {
+    case 1:
+        b();
+}
+            `,
+            options: [{ reportUnusedFallthroughComment: true }]
+        },
```

Comment:
i think we should add this case in the docs `correct` section where `/* fallthrough */ comments are allowed to show they don't permit fallthrough over cases

Reply:
I would like to gently push back on this. As a user, I find this particular example more confusing than illustrative. It deals with an edge case implementation detail that's honestly a judgement call that could go either way, rather than anything to do with the primary purpose of the rule. (i am happy to unpack this position more if you'd like, or suggest an alternative example to add)

That being said, feel free to just say "overruled" and I will happily put it there for you :)

Reply:
i just thought this would inform the users that comments inside `/* */` will not premit the fallthrough, but if this can create confusion we can leave this idea.

Reply:
There may be a misunderstanding. Comments inside `/* */` are checked the same as `//`:

```js
        {
            code: `
switch (foo) {
    default:
        a();
        break;
    /* falls through */
    case 1:
        b();
}`,
            options: [{ reportUnusedFallthroughComment: true }],
            errors: [
                {
                    line: 6,
                    messageId: "unusedFallthroughComment"
                }
            ]

```


The reason the case this comment thread is on doesn't flag is that the "subsequent case" is part of a different `switch` statement entirely.

Does that address the confusion or am I missing what you're saying?

Reply:
Ya my bad sorry for the misunderstanding.